### PR TITLE
Fix XMLBearTest

### PR DESCRIPTION
**Pull request by Makman2**
Fixes https://github.com/coala/coala-bears/issues/2052 and https://github.com/coala/coala-bears/issues/2053

### CODE REVIEW

File: tests/xml2/XMLBearTest.py [hunk 1]
```diff
@@ -8,12 +8,17 @@
...
+from coala_utils.string_processing import escape
 
 
-def load_testdata(filename):
-    path = os.path.join(os.path.dirname(__file__),
+def get_testfile_path(testfile):
+    return os.path.join(os.path.dirname(__file__),
                         'test_files',
-                        filename)
+                        testfile)
```
[the remark this answers is not in the document]
Comment (Makman2) [reply tests/xml2/XMLBearTest.py]: oh right, will do so :+1:

**Comment by Makman2** [comment]
Alright will separate into separate commits :+1: I already separated (1) and (3), but wasn't sure in the end whether everything should be put into a single commit to fix the CI immediately with that.

**Comment by jayvdb** [comment]
ack 65ed64e 81a417e baa8de1

**Comment by Makman2** [comment]
reack b864b6c a080829 255aa7a

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**